src/Management/src/OpenTelemetryBase/Steeltoe.Management.OpenTelemetryBase.csproj

```diff
@@ -0,0 +1,22 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="..\..\..\..\versions.props" />
+  <PropertyGroup>
+    <Description>Steeltoe Management OpenTelemetry</Description>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <AssemblyName>Steeltoe.Management.OpenTelemetryBase</AssemblyName>
+    <PackageId>Steeltoe.Management.OpenTelemetryBase</PackageId>
+    <PackageTags>Tracing;OpenTelemetry;Management;Monitoring</PackageTags>
+    <RootNamespace>Steeltoe.Management.OpenTelemetry</RootNamespace>
+  </PropertyGroup>
+  <Import Project="..\..\..\..\sharedproject.props" />
+
+  <ItemGroup>
+    <PackageReference Include="OpenTelemetry" Version="$(OpenTelemetryVersion)" />
+    <PackageReference Include="OpenTelemetry.Exporter.Zipkin" Version="$(OpenTelemetryVersion)" />
+  </ItemGroup>
+
+  <ItemGroup>
```

Comment:
use `Condition="'$(CI_BUILD)' == 'True'"` to make this conditionally a package or project reference

Reply:
done